.readthedocs.yaml

```diff
@@ -0,0 +1,32 @@
+# .readthedocs.yaml
```

Comment:
We should link to the new URL for docs in the README.

Reply:
I'll do another patch to update the links (in the pyproject.toml file too) once the docs will online on readthedocs.